src/main/java/com/google/sps/classes/Utility.java

```diff
@@ -125,4 +129,28 @@ public static void addNewUser(String firstName, String lastName, String username
       logger.log(Level.SEVERE, exception.getMessage(), exception);
     }
   }
+
+  /** 
+   * Create a question object using the results from a query.
+   */
+  public static QuestionObject buildQuestion(ResultSet queryResult) {
```

Comment:
I know it doesn't have to do specifically with this PR, but do you think changing the name of `QuestionObject` to just `Question` would be a better idea? I think the `Object` is redundant.

Reply:
yep, I did it in another branch, thanks!